designs/l1_l4_security/l1_l4_security.md

```diff
@@ -0,0 +1,75 @@
+# L1-L4 security leveraging OpenStack security groups
+This document describes the notifications available for OpenStack security groups
+
+## Assignees
+Intel's OSC core team
+
+## Background
+OSC will support single page view to manage OpenStack security groups. User will be able to manage the OpenStack security groups and security group rules from OSC.
+
+## Constraints and Assumptions
+No addtional security groups are configured by user or admin directly on OpenStack.
```

Comment:
>admin policies to restrict the user from creating the security groups
I think while the that point clarifies that we intend to restrict creating security groups(clarify it create/edit/delete), we should still add that the OSC synchronization jobs which run based on notifications or with a set interval will get the security groups back in conformance by making sure whatever is configured in OSC is what is reflected in openstack